cfme/tests/ansible/test_embedded_ansible_basic.py

```diff
@@ -150,6 +150,29 @@ def test_embedded_ansible_repository_crud(ansible_repository, wait_for_ansible):
     assert view.description.value == updated_description
 
 
+@pytest.mark.rhel_testing
+@pytest.mark.tier(1)
+def test_embedded_ansible_repository_invalid_url_crud(appliance, wait_for_ansible):
+    """
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: critical
+        initialEstimate: 1/6h
+        tags: ansible_embed
+    """
+    repositories = appliance.collections.ansible_repositories
+    repository = repositories.create(
```

Comment:
Rather than taking ```appliance``` argument for little use you can think of accesing it by this way: ```wait_for_ansible.appliance.collections.ansible_repositories```

Reply:
@ganeshhubale we can get appliance object from other collection object but here `wait_for_ansible` is returning server role enable status not any collection object.